docs/docs/deploy/project-errors.md

```diff
@@ -23,37 +23,21 @@ Before diving into troubleshooting, it's helpful to understand Rill's error mana
-2. **Timeouts, OOM** - Assuming the data in Rill Cloud is a lot larger, timeouts and OOM issues can occur. [Contact us](/contact) if you see any related error messages.
-
-To troubleshoot model errors:
-
-1. **Check the model's status** in the [project status page](/manage/project-management#checking-deployment-status)
-2. **Review the error message** for specific details about what failed. Common error messages and their solutions:
-   - **`Failed to connect to ...`**: Issue with your connector. If it worked locally, check your production credentials and [firewall settings](/build/connectors/data-source#externally-hosted-services)
-   - **`Table with name ... does not exist!`**: Verify the table exists by running `rill query --sql "select * from {table_name} limit 1" --project {project_name}` or `rill project tables --project {project_name}`
-   - **`IO Error: No files found that match the pattern...`**: Check that your production cloud storage folder path is correct and files exist
-   - **`some partitions have errors`**: Run `rill project refresh --model {model_name} --errored-partitions` from your authenticated CLI
-   - **`Out of Memory Error: ...`**: [Contact support](/contact) for assistance with memory issues 
-
-If after going through the above steps, you are still unable to resolve the issue, [contact us!](/contact)
-
-## Metrics View / Dashboard Errors
-
-Metrics view and dashboard errors often stem from issues with the underlying models or configuration problems. Common issues include:
+2. **Timeouts, OOM** - Production data volumes may be larger than local development data, leading to timeouts and out-of-memory issues. [Contact us](/contact) if you see any related error messages.
+3. **Production credentials** - Connector credentials configured for production may differ from local development. Verify your [production credentials](/build/connectors/templating).
 
-- **Model Dependencies:** Dashboards failing because their underlying models have errors
-- **Missing Dimensions/Measures:** References to fields that don't exist in the underlying model
+To troubleshoot deployment errors:
 
-To resolve metrics view and dashboard errors:
+1. **Check the resource status** in the [project status page](/manage/project-management#checking-deployment-status)
+2. **Review project logs** using `rill project logs` or the Rill Cloud UI
+3. **Compare with local behavior** - If it worked locally, check production-specific configuration differences
 
-1. Verify that all referenced models are building successfully
-2. Check the measures and dimensions in your metrics YAML in GitHub or Rill Developer matches an existing column in your data
-3. Check that any changes to the metrics dimensions and measures are reflected in the explore YAML.
+For detailed troubleshooting steps and common error solutions, see the [Debugging Rill Projects](/build/debugging#troubleshooting-common-errors) guide.
```

Comment:
This CTA also exists at the top of the page. To avoid repetition, we should include it only once on this page.